src/main/java/de/blablubbabc/paintball/gadgets/handlers/MineHandler.java

```diff
@@ -133,11 +133,11 @@ protected void onBlockPlace(BlockPlaceEvent event, Match match) {
 
 							@Override
 							public void run() {
-								block.setBlockData(Material.FLOWER_POT.createBlockData());
+								block.setBlockData(getItemType().createBlockData());
```

Comment:
There is no way to actually configure the non-default item type, right? Also, the special pressure plate behavior you mentioned is not actually part of the changes.

Reply:
You should be able to change the item type in the `Shop` section at the end of the config. And yeah, I didn't include the pressure plate listener because I was trying to keep the changes to a minimum as you said, and it's super simple to implement in an addon.